en/docs/reference/connectors/salesforce-connectors/salesforcebulk-v2-connector-example.md

```diff
@@ -84,7 +114,9 @@ Lets add the operations to the resources in `Salesforce` API
         3. Provide your Salesforce connected app's client id in the `Client ID` text box.
         4. Provide your Salesforce connected app's client secret in the `Client Secret` text box.
         5. Provide your Salesforce connected app's refresh token in the `Refresh Token` text box.
-        6. Provide your Salesforceconnected app's Access Token in the `Access Token` text box.
+        6. (Not Recomendded) Provide your Salesforceconnected app's Access Token in the `Access Token` text box.
```

Comment:
Shall we remove the (Not recommended) part? 

And let's add a line saying we recommend not to use access token due to following reason. and put next two points as the explanation.